spec/params_spec.cr

```diff
@@ -0,0 +1,96 @@
+require "./spec_helper"
+
+private class TestOperationWithDefaultParamKey < Avram::Operation
+  attribute title : String
+
+  def run
+  end
+end
+
+private class TestOperationWithCustomParamKey < Avram::Operation
+  param_key :test_op
+  attribute title : String
+
+  def run
+  end
+end
+
+private class SaveUser < User::SaveOperation
+end
+
+private class NestedParams
+  include Avram::Paramable
+
+  @data : Hash(String, String) = {} of String => String
+
+  def initialize(@data)
+  end
+
+  def nested?(key : String) : Hash(String, String)
+    if @data.keys.map(&.split(':').first).includes?(key)
+      @data
+    else
+      {} of String => String
+    end
+  end
+
+  def nested(key : String) : Hash(String, String)
+    nested?(key)
+  end
+
+  def get?(key)
+    raise "Not implemented"
+  end
+
+  def get(key)
+    raise "Not implemented"
+  end
+
+  def nested_file?(key)
+    raise "Not implemented"
+  end
+
+  def nested_file(key)
+    raise "Not implemented"
+  end
```

Comment:
Not necessary to do now at all, but since we do these "fake" params in at least 2 places now, maybe we could have an `Avram::FakeParams` abstract class in `spec_support` that adds these methods in so you don't have to duplicate it.

Up to you though!